src/com/github/johnthagen/cppcheck/CppcheckError.java

```diff
@@ -0,0 +1,8 @@
+package com.github.johnthagen.cppcheck;
+
+public class CppcheckError extends Error {
+    public CppcheckError(String message)
```

Comment:
Minor, but for consistency let's make everything immutable unless it needs to be mutable.

```java
 public CppcheckError(final String message)
```

Reply:
Will do - it's been a while since I did Java on a daily basis. So if the IDE doesn't warn about it I will probably miss such stuff for a while :D

Reply:
done